torch/distributed/distributed_c10d.py

```diff
@@ -1559,11 +1580,12 @@ def new_group(ranks=None, timeout=default_pg_timeout, backend=None):
                                    backend,
                                    default_store,
                                    timeout=timeout)
+    gLogger.info(f"type in new_group {type(pg)}")
```

Comment:
The call chain is a spawned process ==> `new_group()` ==> `_new_process_group_helper()`.
The funny logs are 
(from running `python test/distributed/test_ddp_with_rpc.py TestDdpWithRpc.test_setup_no_ddp`)
```
2020-02-24 20:08:35,139 distributed_c10d.py:491 INFO p:71166 t:139837655197504: Creating a gloo group <class 'torch.distributed.ProcessGroupGloo'>
2020-02-24 20:08:35,140 distributed_c10d.py:508 INFO p:71166 t:139837655197504: Created a gloo group <class 'torch.distributed.ProcessGroupGloo'>
2020-02-24 20:08:35,145 distributed_c10d.py:1583 INFO p:71166 t:139837655197504: type in new_group <class 'object'>
2020-02-24 20:08:35,145 distributed_c10d.py:1590 INFO p:71166 t:139837655197504: type before return in new_group <class 'object'>

```